lib/middleware/token.js

```diff
@@ -52,15 +54,15 @@ var TokenError = require('../errors/tokenerror');
  */
 module.exports = function token(server, options) {
   options = options || {};
-  
-  if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.token}} middleware requires a server argument')); }
```

Comment:
Forget the bikeshedding here. A user who actually see's this and feels it's valid will create an issue for us to fix it later. Anyways, my thoughts:

```
module.exports = function token(server, options) {
```

`server` is the arg here, so don't translate it because the user who reads `serveur` will never be able to map it back in the code if you translate it without translating the variable name (which we're not doing).